### PR TITLE
Mdeploy

### PR DESCRIPTION
**Pull request by squanchd**
AKS deployment support

### CODE REVIEW

File: src/mdeploy.js [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+    {
+      message: 'Password:',
+      type: 'password',
+      name: 'password'
+    }
+  ])
+
+  const { username, password } = credentials
+
+  shell.exec(`az login -u ${username} -p ${password}`)
```
Comment (polygonhell) [inline src/mdeploy.js]: Why require they have access to the azure account in addition to credentials for the k8's cluster?

File: src/mdeploy.js [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+      choices: aksServices
+    }
+  ]
+
+  const aksServiceAnswer = await prompt(aksServiceQuestion)
+  const { aksService } = aksServiceAnswer
+
+  console.log(chalk.blueBright('Getting AKS credentials'))
+  shell.exec(
+    `az aks get-credentials --resource-group ${resourceGroup} --name ${aksService} --override`
```
Comment (polygonhell) [inline src/mdeploy.js]: The require contributor permistons for the Azure account, they could be provided with K8's credentials independently

File: src/mdeploy.js (hunk 1 above)
Comment (squanchd) [reply src/mdeploy.js]: These are two seperate paths - one that uses the AZ utilities and one the doesn't.

File: src/mdeploy.js (hunk 2 above)
Comment (squanchd) [reply src/mdeploy.js]: Similar to the previous comment, this is seperate from the path using the K8 configuration path. It is particular to the Azure path.

File: src/mdeploy.js [hunk 3]
```diff
@@ -0,0 +1,307 @@
...
+          { name: 'Update', value: 'update' }
+        ]
+      }
+    ]
+
+    const answers = await prompt(deleteAndDeployOrUpdateQuestions)
+
+    switch (answers.deleteAndDeployOrUpdate) {
+      case 'delete':
+        shell.exec(
```
Comment (dlewissandy) [inline src/mdeploy.js]: Just to confirm, shell.exec is synchronous by default?

Comment (squanchd) [reply src/mdeploy.js]: yes.

**Comment by squanchd** [comment]
Is any further action required for this or can this be merged to _develop_? @polygonhell @witt3rd

**Pull request merged**